understanding/20/headings-and-labels.html

```diff
@@ -30,6 +30,9 @@ <h2>Intent of Headings and Labels</h2>
       <p>Labels and headings do not need to be lengthy. A word, or even a single character,
          may suffice if it provides an appropriate cue to finding and navigating content.
       </p>
+       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive label without additional text. In these cases, authors should ensure that the image and its use as label in a particular context can be considered conventional and widely understood. 
+         Note that the same icon can be interpreted differently in different contexts and still be considered descriptive in that context. For example, placed after a search field, a loupe icon is commonly interpreted as triggering a search. 
+         Placed on an image, a loupe icon is commonly interpreted as opening a magnified version of the image (or zooming into that image).</p>
```

Comment:
take or leave these suggestions.  just trying to expand on the idea here.
```suggestion
         Placed on or near another image, a loupe or magnify glass icon is commonly interpreted as a means to view a magnified version of the image (for instance, acting as a mechanism to zoom into the image, or opening a full sized image in a new window).</p>
```

Reply:
outdated due to inflight conflict so redid my suggestion